make/Docs.gmk

```diff
@@ -655,7 +659,7 @@ ifeq ($(ENABLE_PANDOC), true)
             FILES := $f, \
             DEST := $(DOCS_OUTPUTDIR)/specs/, \
             CSS := $(GLOBAL_SPECS_DEFAULT_CSS_FILE), \
-            OPTIONS := -V include-before='$(SPECS_TOP)' -V include-after='$(SPECS_BOTTOM_$($m_$f_NOF_SUBDIRS))', \
+            OPTIONS := $($m_$(notdir $f)_TOC) -V include-before='$(SPECS_TOP)' -V include-after='$(SPECS_BOTTOM_$($m_$f_NOF_SUBDIRS))', \
```

Comment:
I'd like you to calculate a variable for this, a few lines above (can't mark it in github review mode 😞), like:
```
        $(eval $m_$f_OPTIONS := $($m_$(notdir $f)_OPTIONS) \
```

and then use that variable instead in the OPTIONS line.

Reply:
Thanks for the advice; will do.